flixel/system/frontEnds/DebuggerFrontEnd.hx

```diff
@@ -155,8 +159,11 @@ class DebuggerFrontEnd
 			FlxG.stage.stageFocusRect = false; // don't show yellow focus rect on flash
 			FlxG.stage.focus = FlxG.game;
 		}
+		
+		visible = Value;
+		visibilityChanged.dispatch();
```

Comment:
We shouldn't always dispatch the signal, only when the value actually _changed_.